react-native/training/src/app/screens/TodoList/components/Input/index.js

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react';
+import { TextInput } from 'react-native';
+import PropTypes from 'prop-types';
+
+import styles from './styles';
+
+class Input extends Component {
+  state = {
+    text: ''
+  };
+
+  onChangeText = text => {
+    this.setState({ text });
+  };
+
+  onSubmitEditing = () => {
+    const { onSubmit } = this.props;
+    const { text } = this.state;
+
+    if (!text) return;
```

Comment:
It would be good if you show some error here. you can use `Alert`

Reply:
# iOs
![list_ios](https://user-images.githubusercontent.com/40371687/48139741-369b5280-e275-11e8-8c87-2489a5ee90eb.gif)

# Android
![list_android](https://user-images.githubusercontent.com/40371687/48139753-3ef38d80-e275-11e8-932f-db6cb524f143.gif)